### ISSUE TITLE
Register Hub-to-Login Loop Error

### ISSUE THREAD

**Issue by colecci** [tixte/version-2.0#2]
When a user registers an account there is a bug that will popup every once in a while and get the user stuck in a loop so the minute they create their account, they are redirected to the Tixte Hub, then to the login, then hub...etc...

If anyone has any ideas on this, please share your ideas! You could receive a Stage:2 Developer Solution Badge to your Tixte account.

**Comment by michaelfm1211** [comment]
I can confirm this happens on Safari

**Comment by michaelfm1211** [comment]
I have observed that the reason this is happening is that the localStorage.tixte_uname is set to null for some reason when an email account is created, then the user is redirected to the hub, but the hub redirects the user to the login because localStorage.tixte_uname is set to null, but the login redirects the user to the hub because the user is still logged in(via firebase).

**Comment by michaelfm1211** [comment]
This problem has been resolved. I copied the verification code from version-1.0 and added in code to set localStorage.tixte_uname to the user's email. This issue was fixed in commit ```4fbfb6acd0acbe74fdde046dc942220c4e576495```. Please deposit the Stage: 2 Developer Badge into the Tixte account Michael M. with GitHub Login. You may verify my identity because I signed this commit with my GPG key ```38737636CEB4F70B499A4B5564F75D2E266B5A99``` with the name michaelfm1211. Here is the ```git log --show-signature``` output for this commit:
```
commit 4fbfb6acd0acbe74fdde046dc942220c4e576495 (HEAD -> master, origin/master, origin/HEAD)
gpg: Signature made Mon Sep 30 11:09:38 2019 CDT
gpg:                using RSA key 38737636CEB4F70B499A4B5564F75D2E266B5A99
gpg: Good signature from "michaelfm1211 <42911957+michaelfm1211@users.noreply.github.com>" [ultimate]
Author: michaelfm1211 <42911957+michaelfm1211@users.noreply.github.com>
Date:   Mon Sep 30 11:09:37 2019 -0500

    Attempt to Fix Login Loop by Setting localStorage.tixte_uname to uname.value When User is Authorized for Email Login
```
